acceptance-tests/src/test/resources/features/site/footer.feature

```diff
@@ -4,32 +4,32 @@ Feature: Footer
     and conditions and other legal and contact information
 
 
-    Scenario: Terms and conditions
-        Given I navigate to the "home" page
-        When I click on the "Terms and conditions" link
-        Then I should see a page titled "Terms and Conditions"
-        And I should see the content "1              About these terms ..."
-        When I navigate to the "publication with datasets" page
-        Then I can click on the "Terms and conditions" link
-        And I should see a page titled "Terms and Conditions"
-
-    Scenario: Accessibility help
-        Given I navigate to the "home" page
-        When I click on the "Accessibility help" link
-        Then I should see a page titled "Accessibility help"
-        And I should see the content "Our site aims to comply with the World Wide Web Consortium's (W3C's) Web Accessibility Guidelines ..."
-        When I navigate to the "publication with datasets" page
-        Then I can click on the "Accessibility help" link
-        And I should see a page titled "Accessibility help"
-
-    Scenario: Privacy and cookies
-        Given I navigate to the "home" page
-        When I click on the "Privacy and cookies" link
-        Then I should see a page titled "Privacy and cookies"
-        And I should see the content "This Privacy Notice tells you what to expect when NHS Digital collects personal information ..."
-        When I navigate to the "publication with datasets" page
-        Then I can click on the "Privacy and cookies" link
-        And I should see a page titled "Privacy and cookies"
+#    Scenario: Terms and conditions
```

Comment:
Footer links and the page contents are changed. As we discussed earlier this will be done in the second round of fixing issues related to DW_191. Ticket: https://jira.digital.nhs.uk/browse/DW-227